hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -5301,7 +5301,11 @@ private long replayRecoveredEditsForPaths(long minSeqIdForTheRegion, FileSystem
 
       long maxSeqId;
       String fileName = edits.getName();
-      maxSeqId = Math.abs(Long.parseLong(fileName));
+      if (isSkipErrorEnabled() && fileName.endsWith(HConstants.HREGION_EDITS_CORRUPTED_SUFFIX)) {
```

Comment:
recovered.edits have same problem?

Reply:
OK, let me add a UT for that.

Reply:
> recovered.edits have same problem? or problem only for SplitWALToHFile? Can we have UT for this?

I checked the code, recovered.edits will clear the files at the beginning, so we don't need UT now?